src/modules/markets/entities/market.entity.ts

```diff
@@ -4,7 +4,7 @@ import { DATE } from '@ukef/constants';
 import { Column, Entity, PrimaryColumn } from 'typeorm';
 
 @Entity({
-  name: 'ODS_MARKETS',
+  name: 'CIS_USP_READ_MARKETS',
   schema: 'dbo',
```

Comment:
I'm not sure how we've been using `Entity` `name` and `schema` in MDM up until now, but the TypeORM docs say that the `name` is supposed to be the table name.

(Since this class is for the result of a stored procedure rather than for a table, I don't think it makes sense to have an `Entity` decorator at all but I realise that decision would affect a lot of MDM so we probably don't want to change it at this late stage)

Reply:
@IgnatG to be clear, I'm happy for you to do nothing about this comment - feel free to mark it as resolved if that's your plan

Reply:
yes, that would be my expectation as well - that the name is the table name. the only reason why it changed is to match the name of the stored procedure. The long term plan will be to replace the Stored Procedure with the actual tables. I'll create a ticket and add it to Jira